.travis/deploy.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+tag_version=$1
+
+mkdir ~/.bintray/
+file=$HOME/.bintray/.credentials
+cat <<EOF >$file
+realm = Bintray API Realm
+host = api.bintray.com
+user = $BINTRAY_SNOWPLOW_MAVEN_USER
+password = $BINTRAY_SNOWPLOW_MAVEN_API_KEY
+EOF
+
+cd $TRAVIS_BUILD_DIR
+
+project_version=$(sbt version -Dsbt.log.noformat=true | perl -ne 'print "$1\n" if /(\d+\.\d+\.\d+[^\r\n]*)/' | head -n 1 | tr -d '\n')
+if [ "${project_version}" == "${tag_version}" ]; then
+    sbt protocols/publish
+    sbt client/publish
+    sbt server/publish
+    sbt protocols/bintraySyncMavenCentral
+    sbt client/bintraySyncMavenCentral
+    sbt server/bintraySyncMavenCentral
```

Comment:
as we discussed this approach only works for libraries, her the client, for the server you'll need to go through release-manager, e.g. https://github.com/snowplow/snowplow/issues/3441

Reply:
Actually we don't want the assembly to go to maven.